Front/RozenaWeather front/web.cpp

```diff
@@ -0,0 +1,41 @@
+#include "web.h"
+
+Web::Web(QWidget *parent)
+    : QWidget{parent}{
+    window = new QWidget();
+    window->setWindowTitle("Вартість біткоїна");
+
+    label = new QLabel(window);
+    label->setGeometry(50, 50, 200, 50);
+
+    manager = new QNetworkAccessManager(this);
+
+    const QUrl url("https://api.coindesk.com/v1/bpi/currentprice/BTC.json");
```

Comment:
Contact Rufik for permission to use cryptocurrecy services

Reply:
@NazarMarchuk, I`ve received permission from @rufik_crypto for such use, please reconsider your comment.